Sources/ArcGISToolkit/Components/Authentication/CertificatePickerViewModifier.swift

```diff
@@ -74,15 +84,16 @@ import ArcGIS
         Task.detached {
             do {
                 guard certificateURL.startAccessingSecurityScopedResource() else {
-                    await self.showCertificateImportError(nil)
+                    await self.showCertificateError(CertificateError.couldNotAccessCertificateFile)
```

Comment:
I realize that this code is just being modified, but I noticed that work is being done in the `else` clause of this guard statement, which often times makes code harder to reason about because it doesn't fit the mental model of guard statements. Consider restructuring to use an if…else statement:

```swift
if certificateURL.startAccessingSecurityScopedResource() {
    defer { certificateURL.stopAccessingSecurityScopedResource() }
    let credential: NetworkCredential = try .certificate(at: certificateURL, password: password)
    await self.challenge.resume(with: .continueWithCredential(credential))
} else {
    await self.showCertificateError(.couldNotAccessCertificateFile)
}
```

Reply:
done